### ISSUE TITLE
Switching fron en_CA to fr_CA feels odd

### ISSUE THREAD

**Issue by sjstark** [BKWLD/cloak-i18n#10]
Not sure if this was intentional, but the UX feels a bit weird here to me.

Clicking directly on one of the language codes when CA fr or en is selected does not open the drop down and immediate switches.
<img width="447" alt="Screen Shot 2022-04-19 at 10 41 12 AM" src="https://user-images.githubusercontent.com/6759557/164053962-405d151c-120b-4068-ae66-a6cb5fe6c23e.png">

<img width="487" alt="Screen Shot 2022-04-19 at 10 42 11 AM" src="https://user-images.githubusercontent.com/6759557/164054004-e2dac88c-dad5-4f4e-bb49-42c777c7a1e3.png">

Personally I feel like if a language is selected, only that should be shown and when the dropdown is open, the current country with all languages should be shown as an option.

Let me know if this was something that was an intentional choice after discussions with Design/Strategy.

**Comment by weotch** [comment]
This was just my interpretation of [the Owlet designs](https://github.com/BKWLD/cloak-i18n/issues/2#issuecomment-1100277068), aka, not an explicit direction from Design/Strat.  Yeah, this is good feedback.  It is kind of non-standard isn't it?

**Comment by sjstark** [comment]
Yeah I wonder if a selected language should be more explicit in the Owlet designs (we haven't really set up the other language sites yet so not sure if design has really thought about what it looks like with another language selected)

![Screen Shot 2022-04-19 at 11 10 43 AM](https://user-images.githubusercontent.com/6759557/164058631-9bfbde08-b7ef-4079-8f7e-fcd06c9801c1.png)
